builtin/tfc/config_sourcer.go

```diff
@@ -411,15 +469,23 @@ config {
 		"workspace",
 		"The Terraform Cloud workspace associated with the given organization to read the outputs of",
 		docs.Summary(
-			"The outputs associtaed with the most recent state version for the given workspace",
+			"The outputs associated with the most recent state version for the given workspace",
 			"are the ones that are used. These values are refreshed according to",
 			"refreshInternal, a source field.",
 		),
 	)
 
 	doc.SetRequestField(
 		"output",
-		"The name of the output to read the value of",
+		"Optional: The name of the output to read the value of. All outputs if blank.",
+		docs.Summary(
+			"The name of the output from Terraform Cloud to return.",
+			"Only strings and objects representable as json (i.e. maps and lists)",
+			"are currently supported.",
+			"If unspecified, all outputs from the workspace will be read",
+			"and returned as a map[string]object, and can be referenced as such",
+			"in the waypoint.hcl. See <docs> for more details.",
```

Comment:
Was `<docs>` intended to be a URL?

Reply:
Fixed! Not sure which docs I meant there, but I linked to waypoint-examples.